### DISCUSSION TITLE
Size of photos & videos downloaded is 0 & Telegram says: [420 FLOOD_WAIT_X] - A wait   c  of 2740 seconds is required Error

### DISCUSSION THREAD

**Discussion by BaZZokaX1** [Dineshkarthik/telegram_media_downloader#452] [Q&A]
Hi . i tried to download some media from private channel & i get some files successfully downloaded & another photos & videos get 0 size & the script show this error .. how can i fix it 
 Telegram says: [420 FLOOD_WAIT_X] - A wait   c
 of 2740 seconds is required (caused by
 "auth.ExportAuthorization")
 Traceback (most recent call last):
   File
 "C:\Users\AppData\Local\Programs\Py
 thon\Python38\lib\site-packages\pyrogram\cli
 ent.py", line 878, in get_file
     exported_auth = await self.invoke(
   File
 "C:\Users\AppData\Local\Programs\Py
 thon\Python38\lib\site-packages\pyrogram\met
 hods\advanced\invoke.py", line 79, in invoke
     r = await self.session.invoke(
   File
 "C:\Users\AppData\Local\Programs\Py
 thon\Python38\lib\site-packages\pyrogram\ses
 sion\session.py", line 389, in invoke
     return await self.send(query,
 timeout=timeout)
   File
 "C:\Users\AppData\Local\Programs\Py
 thon\Python38\lib\site-packages\pyrogram\ses
 sion\session.py", line 357, in send
     RPCError.raise_it(result, type(data))
   File
 "C:\Users\AppData\Local\Programs\Py
 thon\Python38\lib\site-packages\pyrogram\err
 ors\rpc_error.py", line 91, in raise_it
     raise getattr(
 pyrogram.errors.exceptions.flood_420.FloodWa
 it: Telegram says: [420 FLOOD_WAIT_X] - A
 wait of 2740 seconds is required (caused by
 "auth.ExportAuthorization")

**Comment by tangyoha** [comment]
Try https://github.com/tangyoha/telegram_media_downloader
